modules/s3/main.tf

```diff
@@ -1,13 +1,14 @@
 locals {
-  function_name = "Coralogix-S3-${random_string.this.result}"
+  function_name = "Coralogix-${var.package_name}-${random_string.this.result}"
   coralogix_regions = {
-    Europe    = "api.coralogix.com"
-    Europe2   = "api.eu2.coralogix.com"
-    India     = "api.app.coralogix.in"
-    Singapore = "api.coralogixsg.com"
-    US        = "api.coralogix.us"
-    Custom    = var.CustomDomain
+    Europe    = "ingress.coralogix.com"
+    Europe2   = "ingress.eu2.coralogix.com"
+    India     = "ingress.coralogix.in"
+    Singapore = "ingress.coralogixsg.com"
+    US        = "ingress.coralogix.us"
+    Custom    = var.custom_domain
```

Comment:
I believe that it should be `ingress.{var.custom_domain}`

Reply:
the user needs to input the entire url for his custom domain, so i not sure that we need to add more things to the url.

Reply:
@oded-dd we didnt want limit the URL so left it without any prefix or restrictions

Reply:
So it should not be custom domain but custom URL